lib/widget/containers/intubation_checklist_container.dart

```diff
@@ -28,7 +30,7 @@ class IntubationChecklistContainer extends StatelessWidget {
           child: ListView(
             children: [
               Container(padding: const EdgeInsetsDirectional.only(top: 10)),
-              ...renderList(),
+              ...__renderList(),
```

Comment:
just one underline in the name please :pray: :slightly_smiling_face:

Reply:
Just removed. Also changed the reusable checklist widget from Stateful to be Stateless due to the use of provider. Learnt a lot from it. Thanks for the guide!